src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -264,11 +275,21 @@ static void checkAttributes(AnonymousFunctionExpressionSyntax syntax, SyntaxList
                     if (!parameter.Identifier.IsMissing)
                     {
                         var thisParameterHasType = parameter.Type != null;
+                        var thisParameterHasDefault = parameter.Default != null;
```

Comment:
>parameter.Default != null [](http://example.com/codeflow?start=54&length=25)

Minor: This could be inlined in the one use below, to avoid getting the default unnecessarily.